omod/src/main/webapp/pages/deliveryNotes.gsp

```diff
@@ -622,38 +637,53 @@
 				</div>				
 			</div>			
 			<div class="clear"></div>
-			
-			
-			
-			
-			
-            
-           
         </fieldset>
+
         <fieldset class="no-confirmation">
             <legend>Treatment</legend>
-            <field>
-                <label>Counselled on feeding options?</label>
-                <label style="padding-left:0px; width: auto;">
-                    <input type="radio" name="concept.5526AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
-                    Yes
-                </label>
-                <label style="padding-left:0px; width: auto;">
-                    <input type="radio" name="concept.5526AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
-                    No
-                </label>
-            </field>
-            <field>
-                <label>Vitamin A supplementation?</label>
-                <label style="padding-left:0px; width: auto;">
-                    <input type="radio" name="concept.161534AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
-                    Yes
-                </label>
-                <label style="padding-left:0px; width: auto;">
-                    <input type="radio" name="concept.161534AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
-                    No
-                </label>
-            </field>
+			<div class="onerow floating-controls hiv-info">
+				<div class="col4" style="width: 33%; margin: 0 1% 0 0">
+					<div class="testbox">
+						<div>Counselled on feeding options?</div>
+						<label>
+							<input type="radio" name="concept.5526AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
+							Yes
+						</label><br>
+						<label>
+							<input type="radio" name="concept.5526AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
+							No
+						</label>
+					</div>
+				</div>
+				<div class="col4" style="width: 33%; margin: 0 1% 0 0">
+					<div class="testbox">
+							<div>Vitamin A supplementation?</div>
+							<label>
+								<input type="radio" name="concept.161534AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
+								Yes
+							</label><br>
+							<label>
+								<input type="radio" name="concept.161534AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" value="1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA">
+								No
+							</label>
+					</div>
+				</div>
+				<div class="col4 last" style="width: 32%;">
+					<div class="testbox anc-results">
+						<div>Vitamin k supplementation?</div>
+						<label>
+							<input type="radio" name="concept.1000542AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
```

Comment:
The correct concept uuid to be used is **42ae6b6b-723f-4d41-b955-67ec99660fe2** NOT what you have supplied, we don't have such uuid in the concept table